curriculum/challenges/english/blocks/workshop-build-a-video-display-using-iframe/68ba9c4f1688914d72876094.md

```diff
@@ -0,0 +1,51 @@
+---
+id: 68ba9c4f1688914d72876094
+title: Step 3
+challengeType: 0
+dashedName: step-3
+---
+
+# --description--
+
+In the first lecture on the `iframe` element, you learned it's a replaced element just like `img`. That means it can also take the `width` and `height` properties to determine how tall and wide it should be.
```

Comment:
```suggestion
In the first lesson on the `iframe` element, you learned it's a replaced element just like `img`. That means it can also take the `width` and `height` properties to determine how tall and wide it should be.
```